features/teacher_views_report_from_the_instructional_materials_page.feature

```diff
@@ -9,8 +9,24 @@ Feature: Teacher views report from the instructional materials page of a class
     And the database has been seeded
     And I am logged in with the username teacher
     And I go to Instructional Materials page for "My Class"
-    
-    
+
+
+  @javascript
+  Scenario: Teacher should be able to toggle names
+    When the following student answers:
+      | student   | class          | investigation       | question_prompt | answer |
+      | dave      | My Class       | Aerodynamics        | c               | y      |
+      | chuck     | My Class       | Aerodynamics        | a               | Y      |
+    And I go to Instructional Materials page for "My Class"
+    And I follow "Run Report"
+    Then Report page should have student name "Dave Doe" in answered section for the question "a"
```

Comment:
I can see and believe it works, just out of curiosity, shouldn't Dave be in section for the question "c"? I'm looking at the table above. :wink: